suggest/preexisting_conditions.txt

```diff
@@ -2288,14 +2260,13 @@ campomelic dysplasia (CD)
 camptodactyly-arthropathy-coxa vara-pericarditis syndrome
 camptodactyly-tall stature-scoliosis-hearing loss syndrome (LS)
 campylobacteriosis
-cancer
+cancer (CA)
```

Comment:
this one seems odd... but maybe I don't have the right background knowledge

Reply:
yeah removed.